file-manager/src/main/scala/it/pagopa/interop/commons/files/service/FileManager.scala

```diff
@@ -20,6 +20,16 @@ trait FileManager {
     path: String
   )(resourceId: String, fileName: String, fileContent: Array[Byte]): Future[StorageFilePath]
 
+  def storeBytes(containerPath: String, path: String, filename: String)(
+    fileContent: Array[Byte]
+  ): Future[StorageFilePath]
+
+  def listFiles(container: String)(prefix: String): Future[List[StorageFilePath]]
```

Comment:
in the generic file manager domain, IMO this could be still called `path`
```suggestion
  def listFiles(container: String)(path: String): Future[List[StorageFilePath]]
```